Modules/Warnings.cs

```diff
@@ -343,6 +343,44 @@ public static string MessageLink(DiscordMessage msg)
             DiscordMessage msg = await ctx.RespondAsync($"{Program.cfgjson.Emoji.Warning} {targetUser.Mention} was warned: **{reason.Replace("`", "\\`").Replace("*", "\\*")}**");
             UserWarning warning = await GiveWarningAsync(targetUser, ctx.User, reason, MessageLink(msg), ctx.Channel);
         }
+        
+        [
+            Command("anonwarn"),
+            Description("Issues a formal warning to a user from a private channel."),
+            Aliases("anonwam", "anonwarm"),
+            HomeServer, RequireHomeserverPerm(ServerPermLevel.TrialMod)
+        ]
+        public async Task WarnCmd(
+            CommandContext ctx,
+            [Description("The user you are warning. Accepts many formats.")] DiscordUser targetUser,
+            [Description("The channel you wish for the warning message to appear in.")] DiscordChannel targetChannel,

```

Comment:
I probably would have put the channel first in these arguments, it makes more sense to me.